shell/platform/tizen/channels/app_control.cc

```diff
@@ -0,0 +1,325 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "app_control.h"
+#include "flutter/shell/platform/tizen/channels/app_control_channel.h"
+
+#include "flutter/shell/platform/common/client_wrapper/include/flutter/event_stream_handler_functions.h"
```

Comment:
Please sort the includes properly.

```suggestion
#include "flutter/shell/platform/common/client_wrapper/include/flutter/event_stream_handler_functions.h"
#include "flutter/shell/platform/tizen/channels/app_control_channel.h"
```